basic-assembly/basic-gradle-plugin/build.gradle

```diff
@@ -37,7 +37,7 @@ gradlePlugin {
     plugins {
         OSMRunner {
             id = 'org.eclipse.sw360.antenna'
-            implementationClass = 'org.eclipse.sw360.antenna.gradle.AntennaBasicGradlePlugin'
+            implementationClass = 'org.eclipse.sw360.antenna.gradle.BasicGradlePlugin'
         }
```

Comment:
Again, BasicGradlePlugin is from my point not the right viewpoint. It should be GradlePlugin, because from viewpoint Antenna, that is what you need to use. That we customize it for the Bosch Use Case with additional workflow steps is a local phenomenon, not the view of the world.